src/components/button/button.js

```diff
@@ -57,7 +57,7 @@ angular
  *  </md-button>
  * </hljs>
  */
-function MdButtonDirective($mdInkRipple, $mdTheming, $mdAria, $timeout) {
+function MdButtonDirective($mdButtonInkRipple, $mdTheming, $mdAria, $timeout) {
```

Comment:
I think I like this approach as opposed to a bunch of boilerplate functions on the original `$mdInkRipple`